### PR TITLE
feat: Eclipse USDC token metadata

### PR DESCRIPTION
**Pull request by daniel-savu**

### CODE REVIEW

File: deployments/warp_routes/USDC/eclipse/metadata.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "USDC",
+  "symbol": "USDC",
+  "description": "Warp Route USDC on Eclipse",
+  "image": "https://github.com/spothq/cryptocurrency-icons/blob/1a63530be6e374711a8554f31b17e4cb92c25fa5/128/icon/usdc.png",
```
Comment (paulbalaji) [inline deployments/warp_routes/USDC/eclipse/metadata.json:5]: should it be <raw.....> as done for SOL?
https://github.com/hyperlane-xyz/hyperlane-registry/blob/4ee77e9bf90eff2f99d66b4c7b941fe6d3da9612/deployments/warp_routes/SOL/eclipse/metadata.json#L5

```suggestion
  "image": "https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/128/icon/usdc.png",
```

Comment (daniel-savu) [reply deployments/warp_routes/USDC/eclipse/metadata.json]: great catch, pushed with the permalink

File: deployments/warp_routes/USDC/eclipse/metadata.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (jmrossy) [inline deployments/warp_routes/USDC/eclipse/metadata.json:1]: I don't really understand what these files are for? They're not warp configs so do they belong here in the warp_routes deployments dir?

**Review by jmrossy — APPROVED** [review]

File: deployments/warp_routes/USDC/eclipse/metadata.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  "name": "USDC",
```
Comment (tkporter) [inline deployments/warp_routes/USDC/eclipse/metadata.json:2]: copying my comment from the monorepo PR:

I'd probably do `USD Coin` to match what is on [Ethereum](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48#readProxyContract#F16) and [Solana](https://solscan.io/token/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v#metadata)

**Review by tkporter — APPROVED** [review]
😎

**Pull request merged**